api/src/pcapi/alembic/versions/20240301T135957_e82695d3ec03_add_price_index_on_collectice_stock.py

```diff
@@ -0,0 +1,35 @@
+"""
+add price index on collective_stock
+"""
+
+from alembic import op
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "e82695d3ec03"
+down_revision = "c8e741468d2a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    with op.get_context().autocommit_block():
+        op.create_index(
```

Comment:
Es-tu sûr que ça passera avec le _timeout_ par défaut ?

Reply:
La création de l'index, dans le staging, prend un peut plus d'une seconde a peine 
Time: 1.317s (1 second), executed in: 1.316s (1 second) très exactement concernant les collective_stock. Du coup, comme c'est 3 minutes ( il me semble ) avant le timeout, il y a largement le temps.